app/models/order.rb

```diff
@@ -11,4 +11,21 @@ def total_price
     order_items.sum(:line_item_total)
   end
 
+  def self.count_by_status
+    Order.group(:status).count
+  end
+
+  def self.sorted_orders(sort_by='all')
+    if sort_by == 'ordered'
+      Order.ordered
+    elsif sort_by == 'paid'
+      Order.paid
+    elsif sort_by == 'cancelled'
+      Order.cancelled
+    elsif sort_by == 'completed'
+      Order.completed
+    else
+      Order.all
```

Comment:
Whoa I think theres a Mod 1 Repo somewhere that looks like this....but refactoring it is hard so we will have to get crafty tomorrow